src/github.com/couchbase/sync_gateway/db/change_cache_test.go

```diff
@@ -689,6 +688,38 @@ func FailingTestChannelRace(t *testing.T) {
 	close(options.Terminator)
 }
 
+// Test retrieval of skipped sequence using view.  Unit test catches panic, but we don't currently have a way
+// to simulate an entry that makes it to the bucket (and so is accessible to the view),
+// but doesn't show up on the TAP feed.  Cache logging in this test validates that view retrieval is working, but
+// the subsequent addition of the item returned by the view call to the cache is treated as a duplicate, as it was already added
+// when it showed up on the tap feed.
+func TestSkippedViewRetrieval(t *testing.T) {
+	base.LogKeys["Cache"] = true
+	base.LogKeys["Cache+"] = true
+	db := setupTestDBWithCacheOptions(t, shortWaitCache())
+	defer tearDownTestDB(t, db)
+	db.ChannelMapper = channels.NewDefaultChannelMapper()
+
+	// Allow db to initialize and run initial CleanSkippedSequenceQueue
+	time.Sleep(10 * time.Millisecond)
```

Comment:
When a change cache gets initialized, it starts a scheduler goroutine that runs CleanSkippedSequenceQueue immediately, then every (c.options.CacheSkippedSeqMaxWait/2) seconds.  This wait is to ensure that initial run completes before we run the test below (which makes an explicit synchronous call).